third_party/xla_client/pjrt_computation_client.cc

```diff
@@ -482,7 +482,8 @@ PjRtComputationClient::ExecuteComputation(
   CreateDataHandlesCounter()->AddValue(datas.size());
 
   auto mwait = std::make_shared<util::MultiWait>(1);
-  auto lockfn = [&, this, device, returned_future, timed]() mutable {
+  auto lockfn = [&, this, device, returned_future = std::move(*returned_future),
```

Comment:
It would be good to assert that `returned_future.has_value()` before invoking this lambda function. Also do we need `mutable` here? Does it crash without?

Reply:
vaugley remember we need because of `returned_future, I can give it another test next week.

Reply:
hmm actually this is not going to be a silent failure, that's why we see the TPU CI failing.. Let me add the check anyway.